lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -128,6 +128,35 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
     end
   end
 
+  defp handle_req("beacon_blocks_by_root/2/ssz_snappy", message_id, message) do
+    with <<24, snappy_blocks_by_root_request::binary>> <- message,
+         {:ok, ssz_blocks_by_root_request} <- Snappy.decompress(snappy_blocks_by_root_request),
+         {:ok, blocks_by_root_request} <-
+           Ssz.from_ssz(ssz_blocks_by_root_request, SszTypes.BeaconBlocksByRootRequest) do
+      ## TODO: there should be check that the `start_slot` is not older than the `oldest_slot_with_block`
+      %SszTypes.BeaconBlocksByRootRequest{body: body} =
+        blocks_by_root_request
+
+      count = length(body)
+
+      count = min(count, ChainSpec.get("MAX_REQUEST_BLOCKS"))
```

Comment:
Nit: we can use a pipe here, to avoid re-assigning `count`
```suggestion
      count = 
        length(body)
        |> min(ChainSpec.get("MAX_REQUEST_BLOCKS"))
```